addOns/replacer/src/main/javahelp/org/zaproxy/zap/extension/replacer/resources/help/contents/replacer.html

```diff
@@ -61,6 +61,30 @@ <h3>Replacement String</h3>
 Hexadecimal bytes are represented with "\x00 - \xFF".
 The literal string "\x00" can be kept with "\\x00".
 
+Special values may be included in replacement strings:
+<table>
+	<tr>
+		<th>Replacement String</th>
+		<th>Description</th>
+	</tr>
+	<tr>
+		<td>{{RINT|[MIN]|[MAX]}</td>
+		<td>
+			Random Integer. <code>MIN</code> and <code>MAX</code> are each optional. 
+			<code>MIN</code> defaults to <code>0</code>.
+			<code>MAX</code> defaults to <code>Integer.maxValue</code>
```

Comment:
Better add the value or link to the JavaDoc.

Reply:
Link to https://docs.oracle.com/javase/8/docs/api/constant-values.html#java.lang.Integer.MAX_VALUE instead.

Reply:
I've moved where this is to accommodate the change in help needed after adding the checkbox to enable/disable the additional processing but I've updated the reference in 25eb03d.